src/components/dialog/_BaseDialog.tsx

```diff
@@ -60,7 +59,6 @@ export const BaseDialog = ({
   themeOverride,
   width,
 }: BaseDialogProps) => {
-  const { aminoTheme } = useAminoTheme();
```

Comment:
This shouldn't be necessary anymore. I tested in dashboard and it works fine. Maybe there is an edge case I missed though. @ckgardner

Reply:
`|| aminoTheme` was used because the dialog is rendered outside the parent `data-theme` attribute. If you fixed that, then this shouldn't be necessary.

Reply:
Yeh the new root hook sets it to the `documentElement` which is impossible(?) to render outside of I believe...